src/main/java/net/fabricmc/tinyremapper/ClassInstance.java

```diff
@@ -148,10 +163,37 @@ public int getMrjVersion() {
 		return mrjVersion;
 	}
 
+	@Override
 	public String getSuperName() {
 		return superName;
 	}
 
+	@Override
+	public String getSignature() {
+		return signature;
+	}
+
+	@Override
+	public MemberHeader getField(String name, String desc) {
+		return this.resolve(MemberType.FIELD, MemberInstance.getFieldId(name, desc, desc == null));
+	}
+
+	@Override
+	public MemberHeader getMethod(String name, String desc) {
+		return this.resolve(MemberType.METHOD, MemberInstance.getMethodId(name, desc));
+	}
```

Comment:
Can you also have a version of `getField` and `getMethod` that does not require desc? Because my implementation in #63 would require this capability. You can use `resolvePartial` for this purpose.

Reply:
I'd call resolving lookup methods resolveField/resolveMethod, compared to getField/getMethod they search super classes/interfaces for the member - not just the class itself.

Reply:
might be better to make desc nullable